lawnchair/src/app/lawnchair/ui/preferences/AppDrawerPreferences.kt

```diff
@@ -115,6 +119,41 @@ fun AppDrawerPreferences() {
                 }
             }
         }
+        val isDeviceSearch = prefs.performWideSearchExperimental.get() && showDrawerSearchBar.state.value
+        ExpandAndShrink(visible = isDeviceSearch) {
```

Comment:
```suggestion
        if (isDeviceSearch) {
```
Expanding and shrinking is unnecessary, because enabling the feature requires you to enable device search in a separate interface (experimental settings)